titus-server-master/src/main/java/com/netflix/titus/master/supervisor/service/SupervisorConfiguration.java

```diff
@@ -0,0 +1,10 @@
+package com.netflix.titus.master.supervisor.service;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+@Configuration(prefix = "titusMaster.supervisor")
+public interface SupervisorConfiguration {
+    @DefaultValue("300000")
+    long getBootstrapTimeoutMs();
```

Comment:
it looks like this is not being used anywhere?

Reply:
Good catch